docs/input.md

```diff
@@ -389,8 +438,25 @@ Options may also contain a `_reveal` property containing rules for when it shoul
 
 ☆.。.:*・°☆.。.:*・°☆.。.:*・°☆.。.:*・°☆
 
+
 ---
 
+# `abutton`
+
+Append a button to an input.
+
+Abutton is the opposite of a magic-button.  Instead of "magic", it uses JavaScript. On its own it does nothing, but, when used with kiln.js, you can listen for the click event and trigger any functionality you'd like using JavaScript. This makes it even more powerful than a Magic Button, only without rules.
+
+### abutton Arguments
+
+* **tooltip** - text that will display in a tooltip. used to explain what each button is doing, so make it concise!
+* **icon** - The name of a [material design icon](https://material.io/tools/icons/?style=baseline).  Will appear as the icon of the button, defaults to check_circle.
+
+### abutton Events that can be attached to Kiln-Input using kiln.js
+
+* **click**	- Emitted when the radio button gains focus.
```

Comment:
This description doesn't seem to match to the event

Reply:
👍